www/index.html

```diff
@@ -686,7 +687,7 @@ <h3>Expert settings</h3>
                         if your platform supports it. &nbsp;[<a id="stop" href="#expertSettingsDiv" class="alert-link">Permanently hide</a>]
                     </div>
                 </div>
-                <iframe id="articleContent" class="articleIFrame" src="article.html"></iframe>
+                <iframe id="articleContent" class="articleIFrame" src="article.html" referrerpolicy="same-origin" sandbox="allow-same-origin allow-scripts allow-modals allow-forms allow-popups allow-downloads"></iframe>
```

Comment:
Which zim requires `referrerpolicy="same-origin"`? As compared to `no-referrer`

Reply:
I was thinking of this as a way to block any referrer calls outside of the app's context / domain, but you're right that no-referrer might be better. I can't think of any situation in which we need to allow referral. Thank you, I'll test.